assets/src/stories-editor/blocks/amp-story-page/edit.css

```diff
@@ -131,7 +131,7 @@ div[data-type="amp/amp-story-page"] .editor-inner-blocks .editor-block-list__lay
 }
 
 .editor-block-list__layout div[data-type="amp/amp-story-page"][data-amp-selected="parent"] .editor-block-drop-zone {
-	height: 583px;
+	height: 603px;
```

Comment:
The drop-zone wasn't covering the full page for some reason, not allowing dragging stuff in the lower 20px, this adjustment is for that.